src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala

```diff
@@ -74,7 +74,7 @@ class GraphAuthDAO extends AuthDAO with GraphDAO {
   }
 
   override def getMaximumAccessLevel(user: RawlsUserRef, workspaceId: String, txn: RawlsTransaction): WorkspaceAccessLevel = {
-    val workspaces = listWorkspaces(user, isTargetWorkspace(workspaceId), txn)
+    val workspaces = getWorkspaceAccessLevels(user, isTargetWorkspace(workspaceId), txn)
```

Comment:
@helgridly The cause for the CI failures was that getMaximumAccessLevel used listWorkspaces, which caused all of the code that checked workspace permissions to break (since it would return NoAccess if not in the realm). getMaximumAccessLevel should NOT use this new logic, so instead of spending time re-implementing everything I took the lazy approach and just copied the old version of listWorkspaces into a new function called getWorkspaceAccessLevels. Since we're destroying all of this code within the next few weeks anyway, I argue it isn't worth my time to further re-work this.